assets/test/e2e/test.js

```diff
@@ -183,6 +199,74 @@ describe('', function() {
       commonTeardown();
     });
 
+    describe('new project', function() {
+      describe('when creating a new project', function() {
+        it('should be able to show the create project page', goToCreateProjectPage);
+
+        it('should successfully create a new project', function() {
+          var project = {
+            name:        'console-integration-test-project',
+            displayName: 'Console integration test Project',
+            description: 'Created by assets/test/e2e/test.js'
+          };
+
+          goToCreateProjectPage();
+          for (var key in project) {
+            setInputValue(key, project[key]);
+          }
+          clickAndGo('Create', '/project/' + project['name'] + '/overview');
+          expect(element(by.cssContainingText("h1", project['displayName'])).isPresent()).toBe(true);
```

Comment:
this will fail now that @spadgett's empty project page changes have gone in, the h1 isnt there in that case 

https://github.com/openshift/origin/pull/4317/files#diff-c76167480ada997df0b3ae76b42ed923R303